package/debian/rules.jammy

```diff
@@ -33,6 +33,7 @@ override_dh_auto_build:
 override_dh_auto_install:
 	package/package
 	mv $(DESTDIR)$(PREFIX)/local/bin/pyk $(DESTDIR)$(PREFIX)/bin/pyk
+	mv $(DESTDIR)$(PREFIX)/local/bin/kit.py $(DESTDIR)$(PREFIX)/bin/kit.py
```

Comment:
This is the key modification. I'm unsure why the build succeeded earlier, maybe the Python build didn't generate `kit.py`?

Reply:
Weird that it didn't break before, but happy it's working now!